core/dbt/config/runtime.py

```diff
@@ -39,6 +39,24 @@
 from .project import Project
 from .renderer import DbtProjectYamlRenderer, ProfileRenderer
 
+from dbt.config.utils import parse_cli_vars
+
+
+def get_project_and_cli_vars_from_args(args, project_dir: str) -> Tuple[Project, Dict[str, Any]]:
```

Comment:
Restoring this method for the purposes of backwards-compatability for testing. Removed in [Remove from_args from CleanTask](https://github.com/dbt-labs/dbt-core/pull/6369/commits/9e36c32cc504dba21790a4ede6ad45e129664fd8)